quinn-proto/Cargo.toml

```diff
@@ -25,6 +25,7 @@ tls-rustls = ["rustls", "webpki", "ring"]
 native-certs = ["rustls-native-certs"]
 
 [dependencies]
+arbitrary = { version = "0.4.5", features = ["derive"] }
```

Comment:
This dependency should be optional. We can then guard the derives directly on whether the dependency is enabled, which might theoretically be convenient for other downstream purposes.

Reply:
:+1: I've moved as an optional dependency behind the `fuzzing` feature. However, I don't fully understand this: 
> We can then guard the derives directly on whether the dependency is enabled

so I can update or change the dependency if this is suggesting a different approach from what I've added

Reply:
I mean use `cfg(feature = "arbitrary")` rather than `cfg(fuzzing)`, and perhaps emit the `fuzzing` feature entirely since it looks like you're using a cfg flag for that (not totally clear on where that gets set)

Reply:
Ah good I follow now, thank you. I've omitted the `fuzzing` feature. 

>  it looks like you're using a cfg flag for that (not totally clear on where that gets set)

This cfg gets set by `cargo-fuzz` - more [info is in their guide](https://rust-fuzz.github.io/book/cargo-fuzz/guide.html#cfgfuzzing)